### PR TITLE
PERA-747 :: Immersve feature work (with feature flag off)

### PR DESCRIPTION
**Pull request by mitsinsar**
## Summary
- Add In Immersve feature work
- Firebase feature flag added so this PR can be merged earlier (by default it is off, on for debug debug builds)

## Testing
- Immersve is helping test the builds

## Additional Notes
- Closes https://algorandfoundation.atlassian.net/browse/PERA-747

### CODE REVIEW

File: versions.gradle [hunk 1]
```diff
@@ -60,6 +60,6 @@ ext {
 
     peraApplication = [
             versionCode: 120,
-            versionName: "5.10.15"
+            versionName: "90.20.747-1"
```
Comment (michaeltchuang) [inline versions.gradle]: should PRs that target `dev` branch use `90.X.X` or keep the `5.X.X` convention?  cc: @ehanoc 

(i think you are trying to create a feature build with this name and then remove this change when PR is ready to merge...this should be scripted in CI)

File: app/src/main/java/com/algorand/android/modules/walletconnect/cards/domain/usecase/GetWalletConnectRequestPreselectedAccountAddresses.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.algorand.android.modules.walletconnect.cards.domain.usecase
```
Comment (yasin-ce) [inline app/src/main/java/com/algorand/android/modules/walletconnect/cards/domain/usecase/GetWalletConnectRequestPreselectedAccountAddresses.kt:13]: Copyright comment is missing in this and other few files. Could you review all the new files and add missing comments, please?

File: app/src/main/java/com/algorand/android/modules/walletconnect/cards/domain/usecase/CacheWalletConnectRequestPreselectedAccountAddressesUseCase.kt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    private fun getSelectedAccountAddress(wcRequestUrl: String): String? {
+        return wcRequestUrl.split("?")
+            .lastOrNull()
+            ?.split("&")
+            ?.firstOrNull { it.contains(SELECTED_ACCOUNT_QUERY_KEY) }
+            ?.split("=")
+            ?.lastOrNull()
+    }
```
Comment (yasin-ce) [inline app/src/main/java/com/algorand/android/modules/walletconnect/cards/domain/usecase/CacheWalletConnectRequestPreselectedAccountAddressesUseCase.kt]: This logic looks a bit complex. Could you create a test to make sure this is working in all scenarios?

File: app/src/main/java/com/algorand/android/modules/card/CardsViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import kotlinx.coroutines.launch
+
+@HiltViewModel
+class CardsViewModel @Inject constructor(
+    private val getAuthorizedAddressesWebMessage: GetAuthorizedAddressesWebMessage,
+    private val getDeviceIdWebMessage: GetDeviceIdWebMessage,
+    private val parseOpenSystemBrowserUrl: ParseOpenSystemBrowserUrl
+) : BasePeraWebViewViewModel() {
+
+    private val _cardsPreviewFlow = MutableStateFlow<CardsPreview>(CardsPreview())
```
Comment (yasin-ce) [inline app/src/main/java/com/algorand/android/modules/card/CardsViewModel.kt:38]: If I'm not mistaken we are not handling the load url error state here. Could you make sure we are handling the error case correctly?

**Review by yasin-ce — CHANGES_REQUESTED** [review]

**Comment by michaeltchuang** [comment]
@mitsinsar - can you add some testing instructions to PR description for reviewers when you get the chance?  Just replace placeholder text

File: versions.gradle (hunk 1 above)
Comment (yasin-ce) [reply versions.gradle]: Hi @michaeltchuang, we updated the bitrise flows and @mitsinsar reverted that the version change. So we can mark this as resolved.

**Review by yasin-ce — APPROVED** [review]

**Comment by michaeltchuang** [comment]
@yasin-ce i don't have access to bitrise yet to check logs, what's the error there?  i assume we can safely ignore?

**Review by 1atAlcone — APPROVED** [review]

**Pull request merged**